ingress/src/main/java/com/redhat/developer/ingress/RegisterCloudEventModuleCustomizer.java

```diff
@@ -0,0 +1,16 @@
+package com.redhat.developer.ingress;
+
+import javax.inject.Singleton;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import io.cloudevents.jackson.JsonFormat;
+import io.quarkus.jackson.ObjectMapperCustomizer;
+
+@Singleton
+public class RegisterCloudEventModuleCustomizer implements ObjectMapperCustomizer {
```

Comment:
This would be used in combination with my comment about making CloudEventUtil a managed bean in the CDI context. We'd keep the registration of the CloudEvent Jackson Module in a single location.